CHANGELOG/CHANGELOG-3.4.md

```diff
@@ -12,6 +12,9 @@ Previous change logs can be found at [CHANGELOG-3.3](https://github.com/etcd-io/
 - Improve [mvcc: push down RangeOptions.limit argv into index tree to reduce memory overhead](https://github.com/etcd-io/etcd/pull/15137)
 - Improve [server: set multiple concurrentReadTx instances share one txReadBuffer](https://github.com/etcd-io/etcd/pull/15195)
 
+### Package `clientv3`
+- Fix [etcd might send duplicated events to subscribers](https://github.com/etcd-io/etcd/pull/15275).
```

Comment:
Naming for me is off. I don't think we ever refer to Watch API as a publisher subscriber.